### PR TITLE
Seedlist scaffold

### PR DESCRIPTION
**Pull request by rajumsys**
Opening this for collaboration to avoid too many merge conflict

How to Test:
- Start seedlist locally and point to it on openresty
- In current codebase, the endpoint is `/api/v1/inbox-placement/seedlist`; rename it to `/api/v1/inbox-placement/seeds` (according to api doc). 
- Unless you've working project, you can modify `controller.getSeedlist` (in `resources/controller.js`) to return a  list of addresses

Notes: 
- Depends on https://github.com/SparkPost/accusers-api/pull/675/files to use the account option. Account option visibility is set wrong
- Need to confirm path https://github.com/SparkPost/sparkpost-admin-api-documentation/pull/356#discussion_r308461529

### CODE REVIEW

File: src/config/navItems.js [hunk 1]
```diff
@@ -106,36 +108,7 @@ export default [
     icon: Language
   },
   {
-    label: 'Settings',
```
Comment (21042jim) [inline src/config/navItems.js]: why are we moving the settings nav config?

File: src/config/navItems.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
-import { Home, InsertChart, Code, MailOutline, Language, Settings, Compare, NotificationsActive, Signal, VerifiedUser } from '@sparkpost/matchbox-icons';
+import { Home, InsertChart, Code, MailOutline, Language, Compare, NotificationsActive, Signal, VerifiedUser } from '@sparkpost/matchbox-icons';
 
 import campaignNavItems from './campaignNavItems';
+import settingsNavItems from './settingsNavItems';
+import seedlistNavItems from './seedlistNavItems';
```
Comment (21042jim) [inline src/config/navItems.js]: should this be `inboxPlacementNavItems` to be consistent?

Comment (rajumsys) [reply src/config/navItems.js]: That works

File: src/config/navItems.js (hunk 1 above)
Comment (rajumsys) [reply src/config/navItems.js]: due to eslint max line warning

File: src/reducers/index.js [hunk 3]
```diff
@@ -93,7 +94,8 @@ const appReducer = combineReducers({
   typeahead,
   users,
   webhooks,
-  websiteAuth
+  websiteAuth,
+  inboxPlacement
```
Comment (kchu1337) [inline src/reducers/index.js]: Let's move this so it's alphabetical. Easier to find with Redux Devtools.

File: src/config/routes/inboxPlacement.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+// import inboxPlacementPages from 'src/pages/inboxPlacement';
```
Comment (kchu1337) [inline src/config/routes/inboxPlacement.js]: For the other Pages, we added them to `src/pages/index.js` and imported from there

File: src/config/settingsNavItems.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    {
+      label: 'SMTP Settings',
+      to: '/account/smtp'
+    },
+    {
+      label: 'Sending Domains',
+      to: '/account/sending-domains'
+    },
+
```
Comment (kchu1337) [inline src/config/settingsNavItems.js]: Not big deal, but might as well remove this extra line

File: src/helpers/conditions/account.js [hunk 6]
```diff
@@ -23,4 +23,4 @@ export const hasOnlineSupport = ({ account }) => _.get(account, 'support.online'
 export const hasUiOption = (option) => ({ account }) => _.has(account.options, `ui.${option}`);
 export const isAccountUiOptionSet = (option, defaultValue) => ({ account }) => Boolean(_.get(account.options, `ui.${option}`, defaultValue));
 export const isSubscriptionPending = ({ account }) => Boolean(account.pending_subscription);
-export const hasAccountOptionEnabled = (option) => ({ account }) => Boolean(account.options[option]);
+export const hasAccountOptionEnabled = (option) => ({ account }) => Boolean(_.get(account.options, option, false));
```
Comment (kchu1337) [inline src/helpers/conditions/account.js]: Are you expecting options to be null or is this just for safety and edge cases?

Comment (rajumsys) [reply src/helpers/conditions/account.js]: looks like `account.options` isn't populated (until response is arrived)

**Pull request merged**